File: calculator.py
import sys
 
operation = sys.argv[1]
num1 = int(sys.argv[2])
num2 = int(sys.argv[3])
 
def calculate(op, a, b):
    if op == 'add':
        return a+b
    if op == 'subtract':
        return a-b
    if op == 'multiply':
        return a*b
    if op == 'divide' and b != 0:
        return a/b
    return 'Unknown operation'
 
result = calculate(operation, num1, num2)
print(f'Result: {result}')

File: calculator.js
const args = process.argv.slice(2);
const operation = args[0];
const num1 = args[1];
const num2 = args[2];
 
function calculate(op, a, b) {
    if (op === 'add') {
        // TODO: RETURN HERE
    }
    if (op === 'subtract') {
        // TODO: RETURN HERE
    }
    if (op === 'multiply') {
        // TODO: RETURN HERE
    }
    if (op === 'divide') {
        // TODO: RETURN HERE
    }
    return 'Unknown operation';
}
 
const result = calculate(operation, num1, num2);
console.log('Result: ' + result);
